src/command_modules/azure-cli-batch/azure/cli/command_modules/batch/tests/recordings/batch_help_full.txt

```diff
@@ -2815,31 +2875,28 @@ Global Arguments
                          examples.
     --verbose          : Increase logging verbosity. Use --debug for full debug logs.
 
-(env) C:\Code\Forks\azure-cli>cmd /c az batch job-schedule --help 
+(env) D:\GitHub\azure-cli\src\command_modules\azure-cli-batch\azure\cli\command_modules\batch\tests>cmd /c az batch job-schedule --help 
 
 Group
-    az batch job-schedule: Manage your Batch job schedules.
+    az batch job-schedule: Manage Batch job schedules.
 
 Commands:
-    create : Add a job schedule to the specified account.
+    create : Add a Batch job schedule to an account.
     delete : Deletes a job schedule from the specified account.
     disable: Disables a job schedule.
     enable : Enables a job schedule.
     list   : Lists all of the job schedules in the specified account.
-    reset  : Update the properties of the specified job schedule. All updatable properties are
-             replaced with the values specified or reset to default values if unspecified. An
-             updated job specification only applies to new jobs.
-    set    : Update the properties of the specified job schedule. You can independently update the
-             'schedule' and the 'job specification', but any change to either of these entities will
-             reset all properties in that entity.
+    reset  : Reset the properties of a job schedule.  An updated job specification only applies to
+             new jobs.
+    set    : Update the properties of a job schedule.
     show   : Gets information about the specified job schedule.
```

Comment:
I think we're missing some further details on the set/reset behavior here - I like the updated phrasing used above

Reply:
This is probably related to the help text revisions above - either that or it's directly from the swagger spec - so probably not too major.